imagekit-sdk/src/main/java/io/imagekit/sdk/tasks/UrlGen.java

```diff
@@ -95,31 +111,24 @@ private static String buildPathUrl(String path, String urlEndpoint, Map<String,
         }
 
         URI baseUrl=URI.create(urlEndpoint);
-        URI newUri= null;
-        try {
+        URL newUri= null;
             String newPath=baseUrl.getPath();
-            if (tr.toString().equalsIgnoreCase("")){
-                newPath+=path.substring(1,path.length());
-            }
-            else {
-                newPath+=tr+path;
+            if (tr.toString().equalsIgnoreCase("")) {
+                newPath += path.substring(1);
+            } else {
```

Comment:
nit: this is not their code format

Reply:
From what I can tell they don't have a format. Their spacing is all over the place and inconsistent!
removed those spaces anyway.

Reply:
Oh, I actually meant the `} else {' should be:
`} else
{` - I think - I can't see there original code right now - but really doesn't matter.